Formula/m/mono.rb

```diff
@@ -1,18 +1,17 @@
 class Mono < Formula
   desc "Cross platform, open source .NET development framework"
   homepage "https://www.mono-project.com/"
-  url "https://github.com/mono/mono.git",
-      tag:      "mono-6.12.0.206",
-      revision: "0cbf0e290c31adb476f9de0fa44b1d8829affa40"
-  license "MIT"
+  url "https://dl.winehq.org/mono/sources/mono/mono-6.14.0.tar.xz"
```

Comment:
what is the difference between the git tarball vs this one?

Reply:
I had issues with cloning submodules whereas submodules are already cloned in the tarball file, so we don't need to make extra clone from 3rd party dependencies.